src/docker/workflows/confluent-local.ts

```diff
@@ -160,8 +165,60 @@ export class ConfluentLocalWorkflow extends LocalResourceWorkflow {
     progress?: Progress<{ message?: string; increment?: number }>,
   ): Promise<void> {
     this.progress = progress;
-    this.logger.debug("Stopping ...");
-    // TODO(shoup): implement
+
+    this.logAndUpdateProgress(`Checking existing ${this.resourceKind} containers...`);
+    const repoTag = `${ConfluentLocalWorkflow.imageRepo}:${this.imageTag}`;
+    const containerListRequest: ContainerListRequest = {
+      filters: JSON.stringify({
+        ancestor: [repoTag],
+        // label: [MANAGED_CONTAINER_LABEL], // TODO: determine if we want to use this label to filter
```

Comment:
We have some different options here:
- we could allow this extension to stop any `confluentinc/confluent-local` containers as long as the image repo+tag match (current behavior)
- we could lock it down so only extension-started containers can be stopped (which adds friction)
- we could expose a setting that lets users toggle whether or not the extension should be allowed to manage containers without this label and document it, add to notifications, etc.